rasa/server.py

```diff
@@ -1484,6 +1518,18 @@ def _validate_yaml_training_payload(yaml_text: Text) -> None:
         )
 
 
+def _validate_json_training_payload(json_text: Text) -> None:
+    try:
+        RasaYAMLReader
```

Comment:
This appears incomplete 😄 also probably not needed, since you used `rasa.shared.utils.validation.validate_training_data` function

Reply:
ope this is entirely irrelevant sorry